cmd/storage/main.go

```diff
@@ -175,6 +190,37 @@ func connectDB(ctx context.Context) (*sqlx.DB, error) {
 	return sqlx.NewDb(db, "postgres"), nil
 }
 
+func connectMemDB(db *sqlx.DB, errc chan<- error) {
+
+	var repo app.Repo
+	log.Info("USING MEM DB")
+	repo = memdb.New()
+
+	kasse := svckasse.New(cfg.kasse)
+	weather, errWeatherSvc := svcweather.Instance(
+		&svcweather.APIConfig{
+			BaseURL: def.OpenWeatherBaseURL,
+			APIKey:  def.OpenWeatherAPIKey,
+		},
+		&svcweather.APIConfig{
+			BaseURL: def.IpifyBaseURL,
+			APIKey:  def.IpifyAPIKey,
+		})
```

Comment:
My compiler says: 
`cannot use &(svcweather.APIConfig literal) (value of type *svcweather.APIConfig) as *svcweather.APIKeyConfig value in argument to svcweather.Instance`
Please check on your side once more.